python/lsst/meas/algorithms/gaussianPsfFactory.py

```diff
@@ -53,8 +53,8 @@ class GaussianPsfFactory(Config):
         check=isPositive,
     )
     sizeFactor = Field(
-        doc="Kernel size as a factor of fwhm (dimensionless); "
-        + "size = sizeFactor * fwhm; ignored if size is not None",
+        doc = "Kernel size as a factor of fwhm (dimensionless); " +
```

Comment:
I believe spaces around = for keyword arguments are:
- encouraged for keyword arguments that are on their own line
- discouraged for other cases